demos/link-prediction/metapath2vec-link-prediction.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>Unfortunately, the visualisation doesn't look as good with the <code>l2</code>  operator (this was also a problem when working on other similar link prediction notebooks if I remember correctly, might've been CTDNE)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1614/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>